src/App.tsx

```diff
@@ -130,7 +130,7 @@ const Content = () => {
 
         {isTablet ? <FooterMobile /> : <FooterDesktop />}
 
-        <$NotificationsToastArea />
+        <NotificationsToastArea tw="z-[2] [grid-area:Main]" />
```

Comment:
`[grid-area:Main]` seems weird. I thought `[]` was for property value only. Does TW not have a grid-area classname? `grid-area-[Main]`?

Reply:
Yeah no tailwind shortcuts for it. I think it's still pretty readable here though.